mappings/net/minecraft/client/render/entity/EnderDragonEntityRenderer.mapping

```diff
@@ -1,9 +1,37 @@
 CLASS net/minecraft/class_895 net/minecraft/client/render/entity/EnderDragonEntityRenderer
 	FIELD field_21006 EYES_TEX Lnet/minecraft/class_2960;
+	FIELD field_21007 HALF_ROOT_THREE F
+	FIELD field_21008 model Lnet/minecraft/class_895$class_625;
+	FIELD field_21737 DRAGON_SKIN_NO_CULL Lnet/minecraft/class_1921;
+	FIELD field_21738 DRAGON_SKIN_DECAL Lnet/minecraft/class_1921;
+	FIELD field_21739 EYES_LAYER Lnet/minecraft/class_1921;
+	FIELD field_21740 CRYSTAL_BEAM_LAYER Lnet/minecraft/class_1921;
 	FIELD field_4668 CRYSTAL_BEAM_TEX Lnet/minecraft/class_2960;
 	FIELD field_4669 EXPLOSION_TEX Lnet/minecraft/class_2960;
 	FIELD field_4670 SKIN Lnet/minecraft/class_2960;
+	METHOD method_23156 (Lnet/minecraft/class_4588;Lnet/minecraft/class_1159;FF)V
+		ARG 0 vertices
+		ARG 1 matrix
+		ARG 2 y
+		ARG 3 x
+	METHOD method_23157 (Lnet/minecraft/class_4588;Lnet/minecraft/class_1159;I)V
+		ARG 0 vertices
```

Comment:
`vertexConsumer`

Reply:
If it's a VertexConsumer naming it `vertices` is intended.